modules/plugins/jboss-as-7/src/main/java/org/rhq/modules/plugins/jbossas7/BaseComponent.java

```diff
@@ -544,64 +542,12 @@ protected CreateResourceReport deployContent(CreateResourceReport report) {
     public CreateResourceReport runDeploymentMagicOnServer(CreateResourceReport report, String runtimeName,
         String deploymentName, String hash) {
 
-        boolean toServerGroup = context.getResourceKey().contains("server-group=");
```

Comment:
distinction between whether we're creating serverGroup deployment or not is no longer needed - serverGroups don't support adding Deployment child anymore